ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/management-security.yml

```diff
@@ -60,3 +60,43 @@ management_security:
       chain_certificate:
         requirement:
           include_root_ca: true
+  shared_secret_profiles:
+    - profile: profile1
+      secrets:
+        - name: name1
+          secret: $1c$mNvaIXPetcQ3hy7oUnTU3w==
+          receive_lifetime:
+            start_date_time: 12/20/2024 10:00:00
+            end_date_time: 12/20/2025 10:00:00
+          transmit_lifetime:
+            infinite: true
+          local_time: true
+        - name: name0
```

Comment:
Can you please test for 

```
      secrets:
        - name: name
          secret: name_secret
```

Reply:
Made keys receive_lifetime and transmit_lifetime , required. Added test for default secret_type.

        - name: name
          secret: secret
          receive_lifetime:
            infinite: true
          transmit_lifetime:
            infinite: true